test/renderer/plugins/feishuSettings.test.ts

```
@@ -0,0 +1,133 @@
+    <p id="message"></p>
+    <button id="save"></button>
+    <button id="disable"></button>
+    <a id="preset-docs"></a>
+  `
+}
+
+type FeishuSettingsWindow = Window & { deepchatPlugin?: unknown }
+
+const runSettingsScript = async (): Promise<void> => {
+  const script = await readFile(scriptPath, 'utf8')
+  window.eval(`(() => {\n${script}\n})()`)
```

Biome (2.4.15)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)